app/editor/src/features/admin/charts/components/ChartTemplatePreviewOptions.tsx

```diff
@@ -384,22 +384,48 @@ export const ChartTemplatePreviewOptions = () => {
                 }}
               />
             </Row>
-            <Text
-              name="scaleTicksStepSize"
-              label="Step size"
-              width="8ch"
-              type="number"
-              value={chartRequestForm.settings.scaleTicksStepSize?.toString() ?? ''}
-              onChange={(e) => {
-                const value = parseInt(e.target.value);
-                setChartRequestForm({
-                  ...chartRequestForm,
-                  settings: mergeChartSettings(values.settings.options, chartRequestForm.settings, {
-                    scaleTicksStepSize: value,
-                  }),
-                });
-              }}
-            />
+            <Row>
+              <Text
```

Comment:
This will ensure the maximum dataset value + `scaleCalcMax` will result in a chart that expands past the maximum value.